projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/document/write-document-field.html

```diff
@@ -12,7 +12,7 @@
 
   <div style='position:relative'>
 
-    <div [id]="createElementId('fileInputWrapper')" (click)="fileSelectEvent()"></div>
+    <div [id]="createElementId('fileInputWrapper')" (click)="fileSelectEvent()" (keyup)="fileSelectEvent()"></div>
```

Comment:
@olusegz07 Is this change require?

Reply:
Yes, it fixes the issue raised by sonarCloud